test_cromwell.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+shutdown() {
+    cd $INITIAL_DIR
+    pkill -P $$
+    exit 0
+}
+
+trap "shutdown" EXIT
```

Comment:
I had it in there initially but took it out intentionally. If sbt test fails the script would bomb out and not pretty print the results. That could also be trapped I suppose